R/download_data.R

```diff
@@ -193,11 +194,15 @@ download_release_asset <- function(asset_url, path) {
   invisible(path)
 }
 
-auth_url <- function(url) {
+add_auth_header <- function() {
   pat <- Sys.getenv("GITHUB_PAT")
   if (nzchar(pat)) {
-    return(paste0(url, "?access_token=", pat))
+    return(httr::add_headers(Authentication = paste0("token ", pat)))
   }
+  invisible(NULL)
```

Comment:
Would there be any reason to let someone know that they are downloading without authentication? Alternatively, since people don't actually need it why do we have it at all?

Reply:
We have it because it downloading with authentication raises the rate-limiting significantly - which since we only have two datasets it's unlikely anyone would hit that. So maybe we don't need it?

Reply:
I see. I'm inclined to keep it and just add in a comment.